config/default.yml

```
@@ -78,6 +78,8 @@ AllCops:
   # When specifying style guide URLs, any paths and/or fragments will be
   # evaluated relative to the base URL.
   StyleGuideBaseURL: https://rubystyle.guide
+  # Documentation URLs will be constructed using the base URL
```

Can you add period at the end of the sentence?

```suggestion
  # Documentation URLs will be constructed using the base URL.
```